server/server.go

```diff
@@ -30,29 +31,43 @@ import (
 
 const stateFile = "liftbridge"
 
+const (
+	streamsSubject     = "streams"
+	activitySubject    = "activity"
+	activityStream     = activitySubject + "_stream"
+	raftSubject        = "raft"
+	replicationSubject = "replication"
+	acksSubject        = "acks"
+	publishesSubject   = "publishes"
+)
+
 // Server is the main Liftbridge object. Create it by calling New or
 // RunServerWithConfig.
 type Server struct {
-	config             *Config
-	listener           net.Listener
-	nc                 *nats.Conn
-	ncRaft             *nats.Conn
-	ncRepl             *nats.Conn
-	ncAcks             *nats.Conn
-	ncPublishes        *nats.Conn
-	logger             logger.Logger
-	loggerOut          io.Writer
-	api                *grpc.Server
-	metadata           *metadataAPI
-	shutdownCh         chan struct{}
-	raft               atomic.Value
-	leaderSub          *nats.Subscription
-	recoveryStarted    bool
-	latestRecoveredLog *raft.Log
-	mu                 sync.RWMutex
-	shutdown           bool
-	running            bool
-	goroutineWait      sync.WaitGroup
+	config               *Config
+	listener             net.Listener
+	port                 int
+	nc                   *nats.Conn
+	ncRaft               *nats.Conn
+	ncRepl               *nats.Conn
+	ncAcks               *nats.Conn
+	ncPublishes          *nats.Conn
+	ncActivity           *nats.Conn
```

Comment:
I don't understand what this is used for.

Reply:
Hm, that was a previous change that should not be here anymore. Should be fixed in e8f318d.